doc/website/release-notes/iceoryx-unreleased.md

```diff
@@ -1278,3 +1278,15 @@
     ```
 
 57. `iox::posix::getSchedulerPriorityMinimum` and `iox::posix::getSchedulerPriorityMaximum` has become internal API
+
+58. The concurrent queues have gained a `Spsc` or `Mpsc` prefix
+
+    ```cpp
+    // before
+    #include "iceoryx_hoofs/concurrent/lockfree_queue.hpp"
+    iox::concurrent::LockFreeQueue q;
+
+    // after
+    #include "iox/mpmc_lockfree_queue.hpp"
```

Comment:
I would prefer `mpmc` as a separate namespace and stick with the name `LockFreeQueue`. But this is not a hard requirement from my side.